src/analyses/dirty.h

```diff
@@ -20,6 +20,10 @@ Date: March 2013
 #include <util/invariant.h>
 #include <goto-programs/goto_functions.h>
 
+/// Dirty variables are ones which have their address taken so we can't
+/// reliably work out where they may be assigned. All variables of this sort
+/// escape their local scope and are also considered shared state in the
```

Comment:
Nitpick: usually "escape" would mean live longer than the scope, which isn't true here -- they just might be accessed indirectly and/or by callees